index.bs

```diff
@@ -108,6 +110,10 @@ Every {{Document}} has a <dfn for="Document">prerendering browsing contexts map<
     1. [=Queue a global task=] on the [=networking task source=] given |successorBC|'s [=browsing context/active window=] to perform the following steps
 
       1. Set |successorBC|'s [=browsing context/loading mode=] to "`default`".
+
+      1. [=list/For each=] |steps| in |successorBC|'s [=active document=]'s [=Document/post-activation steps list=], run |steps|.
```

Comment:
Do we need to handle exceptions in any particular way here?

Reply:
Hmm, right now the only usage of this construct in this spec cannot throw any kind of an exception, but that's an interesting question in general. I kind of hope that any explicit exception handling could be done in the steps that appear in this list, and that we wouldn't actually have to handle anything ourselves, but I guess that might be too naive.

Reply:
I'll add an assert that the specs don't throw an exception; at least as I'm currently imagining these, we'd never put exception-throwing steps in here.